main/src/cgeo/geocaching/models/Geocache.java

```diff
@@ -1557,12 +1558,14 @@ public boolean addWaypointsFromNote() {
      * @param updateDb if true the added waypoints are stored in DB right away
      * @param namePrefix prefix for waypoint names
      */
-    public boolean addWaypointsFromText(@Nullable final String text, final boolean updateDb, @NonNull final String namePrefix) {
+    public boolean addWaypointsFromText(@Nullable final String text, final boolean updateDb, @NonNull final String namePrefix, final boolean forceExtraction) {
```

Comment:
I do not understand of the parameter forceExtraction.
Why do we need them?

Reply:
`addWaypointsFromText()` is called in two contexts - extracting from listing (needs `forceExtraction = true`) and extracting from personal note (should respect global setting, therefore `forceExtraction = false`).